src/dex-ui-components/Pool/Withdraw.tsx

```diff
@@ -0,0 +1,189 @@
+import { SettingsIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  HStack,
+  Flex,
+  Heading,
+  Spacer,
+  IconButton,
+  Button,
+  Text,
+  TableContainer,
+  Table,
+  Tbody,
+  Tr,
+  Td,
+} from "@chakra-ui/react";
+import { ChangeEvent, useState, useCallback } from "react";
+import { WalletConnectionStatus } from "../models/wallet.model";
+import { TokenInput } from "../TokenInput";
+
+export interface WithdrawProps {
+  walletConnectionStatus: WalletConnectionStatus;
+  poolLpDetails: LPTokenDetails;
+  poolLiquidityDetails: PoolLiquidityDetails;
+  onWithdrawClick: (lpAmount: number) => void;
+  onInputAmountChange?: (lpAmount: number) => void;
+  disableWithdrawButton?: boolean;
+}
+
+interface LPTokenDetails {
+  tokenSymbol: string;
+  userLpAmount: number;
+  userLpPercentage: string;
+}
+
+interface TokenLiquidityDetails {
+  tokenSymbol: string;
+  poolLiquidity: number;
+  userProvidedLiquidity: number;
+}
+
+interface PoolLiquidityDetails {
+  firstToken: TokenLiquidityDetails;
+  secondToken: TokenLiquidityDetails;
+}
+
+interface PoolMetricDisplayProps {
+  label: string;
+  value: string;
+}
+
+const PoolMetricDisplay = (props: PoolMetricDisplayProps) => {
```

Comment:
Note: We should move this to an independent file/component that can be used across the DEX (Swap, Add Liquidity, Withdraw, etc.). This will make it easier for us to propagate style updates in a single file, test the component in isolate (RTL and Storybook), and compose new components that require the metric displays.

Reply:
(no need to change anything right now)